### PR TITLE
Portfolio Email Validation

### PR DESCRIPTION
**Pull request by ifeanyi305**
I just validated the input [type="email"] on my portfolio, and I added the following:

- Markup file (HTML) added
- Cascading  stylesheet file added
- Added javascript file
- Html,CSS and JS linters error fixed
- Descriptive readme file added

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -208,3 +208,19 @@ openFourthModal.addEventListener('click', () => {
...
+const errorMsg = document.querySelector('#email + span.error');
+
+form.addEventListener('click', (event) => {
+  if (mail.value !== mail.value.toLowerCase()) {
+    errorMsg.textContent = 'Email address needs to be in lowercase';
+    errorMsg.className = 'error';
+    event.preventDefault();
+  } else {
+    errorMsg.textContent = '';
+  }
```
Comment (mckent05) [inline script.js]: - According to the project requirement, 'If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent'. Your error message is not displayed near the submit button. Please try to fix this 🙏 

Your project:
error

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @ifeanyi305,

# STATUS: CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your code looks clean 👏 
- Your PR title and summary are descriptive 👏 
- All linter checks are passing ✔️ ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @ifeanyi305, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (ifeanyi305) [reply script.js]: Regarding the error message, I will make the necessary changes to ensure it is displayed near the submit button as required by the project requirements. I will work on this and update you once I have made the changes.

Thank you once again for your time and effort in reviewing my code.

**Comment by ifeanyi305** [comment]
Thank you

**Pull request merged**